src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -272,7 +272,7 @@ extern "C" {
 /**
  * Maximal size of the metadata queue for a fragment
  */
-#define MAX_FRAGMENT_METADATA_COUNT 25
+#define MAX_FRAGMENT_METADATA_COUNT 10
 
 /**
  * Maximum amount of the "tags" element metadata for a fragment

```

Comment:
Can we delete `MAX_FRAGMENT_METADATA_TAGS` or use that instead since it appears it's not being used otherwise ?

Reply:
I did not want to do that since the values are publicly available, some customer might be using them

Reply:
I agree with not removing because it is a public define. But I am also wondering if it would be worth just setting it this way:
`#define MAX_FRAGMENT_METADATA_COUNT MAX_FRAGMENT_METADATA_TAGS ` or, set it the other way round:

`#define MAX_FRAGMENT_METADATA_TAGS MAX_FRAGMENT_METADATA_COUNT`

To make sure in the event customers are using `MAX_FRAGMENT_METADATA_TAGS`, they are reading the right value.

Reply:
Done!